java/query/query-core/src/main/java/sleeper/query/core/tracker/QueryTrackerStore.java

```diff
@@ -18,14 +18,48 @@
 
 import java.util.List;
 
+/**
+ * Interface for tracking the status of a query.
+ */
 public interface QueryTrackerStore {
+    /**
+     * Get the status of a query by query Id.
+     *
+     * @param  queryId               the query Id
+     * @return                       the tracked query
+     * @throws QueryTrackerException if something goes wrong checking the status of the query
+     */
     TrackedQuery getStatus(String queryId) throws QueryTrackerException;
 
+    /**
+     * Get the status of a specific query and sub query.
```

Comment:
This is not true, it just gets the status of the sub-query. The throws tag also needs updating.